reporter-web-app/src/components/IssuesTable.js

```diff
@@ -206,7 +206,7 @@ const IssuesTable = (props) => {
                     <span className="ort-excludes">
                         <Tooltip
                             placement="right"
-                            title={Array.from(webAppPackage.pathExcludeReasons).join(', ')}
+                            title={Array.from(webAppPackage.excludeReasons).join(', ')}
```

Comment:
Commit message: "they show", "This gives"

Reply:
Done